Scripts/Shared.ps1

```diff
@@ -75,9 +75,26 @@ function Setup-NuGet {
     $nugetUrl = $repoConfiguration.Requirements.Nuget.Url
     Invoke-WebRequest -Uri $nugetUrl -OutFile $nugetPath
 
+    Add-ToSystemPath -installPath $installPath
+
     Write-Host " Nuget Installation and configuration completed successfully!"
 }
 
+function Add-ToSystemPath {
+    param([string]$installPath)
+    $path = [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::User)
+
+    if ($path -notlike "*$installPath*") {
+        $newPath = "$path;$installPath"
+        [Environment]::SetEnvironmentVariable("Path", $newPath, [EnvironmentVariableTarget]::User)
+        # Update the current session's PATH environment variable
+        $env:Path += ";$installPath"
+        Write-Host "Path added to user PATH: $installPath"
+    } else {
+        Write-Host "Path already exists in user PATH: $installPath"
+    }
+}
```

Comment:
I dont think we need this as adding the NuGet path to the system environment paths with each new build causes an accumulation of outdated paths, especially after clean builds.

so to avoid this, we should modify our approach to add the NuGet path only to the current PowerShell process environment. This would ensure the path is available during the current session but does not persist, preventing the build-up of obsolete paths.

i would push it to this PR, apart from that, everything looks good to me 🙂